pkg/storage/s3/s3.go

```diff
@@ -414,11 +414,6 @@ func (d *driver) ConfigEnv() (envs envvar.List, err error) {
 		envs = append(envs, envvar.EnvVar{Name: "REGISTRY_STORAGE_S3_KEYID", Value: d.Config.KeyID})
 	}
 
-	if d.Config.ChunkSizeMiB > 4 && d.Config.ChunkSizeMiB < 5121 {
-		chunksize := int64(d.Config.ChunkSizeMiB) * 1024 * 1024
-		envs = append(envs, envvar.EnvVar{Name: "REGISTRY_STORAGE_S3_CHUNKSIZE", Value: chunksize})
-	}
-
```

Comment:
Remind me why are we removing these checks again?

Reply:
was the feature removed or was the featuregate removed? I thought we were just removing the featuregate 🤔